docs/Configuration.rst

```diff
@@ -5380,6 +5380,20 @@ configurable in :ref:`exo.properties <Configuration.ConfigurationOverview>`
     exo.es.search.server.url=http://127.0.0.1:9200
     exo.es.search.server.username=root
     exo.es.search.server.password=xxxxx
+    
+It is also possible to configure the number of search and indexing calls in the http connection pool used by Elasticsearch 
+by defining these parameters:
```

Comment:
Why do we document these parameters differently than others ? It should be consistent IMO and add a short description for each property.

Reply:
I added the description set in the issue